### ISSUE TITLE
Bulk creation of assets

### ISSUE THREAD

**Issue by tehkillerbee** [grokability/snipe-it#10584]
### Snipe-IT Version

latest

### Operating System

na

### Web Server

na

### PHP Version

na

### Is your feature request related to a problem? Please describe. A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]

It takes a long time adding many identical assets, where the only difference is the incrementing serial number/asset tag

### Describe the solution you'd like A clear and concise description of what you want to happen.

It would be nice to be able to bulk create assets - eg. using a base serial number/asset tag and add an incrementing number to it

**Comment by zsmbrvr** [comment]
You can import CSV files. 
There's an IMPORT tab on the left side.

I've added few hundred assets without major issues to the system. Even assigned to users in one go.

**Comment by graceless70** [comment]
How did you git it to generate the new asset numbers for the new assets imported

**Comment by tehkillerbee** [comment]
@zsmbrvr Good point using CSV import but that would almost be more time consuming unless I have to add many, many assets in one go. Especially, since I will have to create the file by hand. I just tried exporting an existing asset to CSV as a starting point and then importing it again, but that cannot be imported directly, unfortunately. It looks like some of the fields contain HTML for some reason.

Also, using CSV file this removes the purpose of the snipe-it GUI, if the user has to resolve to using manual CSV import when adding more than one asset. It also reserves batch adding to "experienced" users. In our company, not all the snipe-it users would be comfortable with using a CSV file.

With that said, it looks like I can omit some of the fields (not all of them are required) so maybe that makes the CSV file simpler to create: https://snipe-it.readme.io/docs/importing

Ideally, I would like to create N assets based on an existing asset. The asset numbers should then be generated incrementally.

@graceless70 I think you will have to write them by hand, if you use the CSV method. The asset tag field is "required" when creating the CSV file

**Comment by HesDead103** [comment]
I'm planning on slapping pre-made labels on everything then scanning into a CSV to import. I was hoping the android app would had one of two options. Let me scan pre-existing asset tags when adding an asset via the app but it does not have a barcode scan option for the asset tag field. The other option I was looking for in the app was just auto generating the asset tag when it is being created but I did not see that as an option. Right now it is looking like I will have to jump through some hoops. First, I have to order pre-made asset tag labels and stick them to all my devices in one go, then I have to scan them and the devices serial numbers into a CSV, then I have to edit the CSV, then I can import the CSV. Definitely a lot more work than if the app just had the ability to scan an asset tag label.
